udata_front/theme/gouvfr/templates/header.html

```diff
@@ -1,5 +1,13 @@
 {% set banner_activated = config['BANNER_ACTIVATED'] %}
+{% set read_only_mode = config['READ_ONLY_MODE'] %}
 {% set next_url = url_for(request.endpoint, **request.view_args) if not request.routing_exception else url_for('site.home') %}
+
+{% if read_only_mode %}
+{% block read_only_banner %}
```

Comment:
Why do we need a block here? 🤔 An include should be enough.

Reply:
Cause I'm a copycat :smile_cat: !